pkg/httpproxy/proxy.go

```diff
@@ -118,5 +119,15 @@ func (p *proxy) proxy(req *http.Request) error {
 	req.URL = destURL
 	req.Header = headerCopy
 
+	if auth != "" {
+		req.Header.Set("Authorization", auth)
+	} else if cAuth != "" {
+		signer := newSigner(cAuth)
+		if signer != nil {
+			return signer.sign(req, p.credentials, cAuth)
+		}
+		req.Header.Set("Authorization", cAuth)
```

Comment:
can we reference auth header? as it's defined in https://github.com/rancher/rancher/pull/17014/files#diff-dbac85ab3314162046444898deb8e4bbR25